src/app/schools/schoolWithRelation.ts

```diff
@@ -23,10 +23,10 @@ export class SchoolWithRelation {
   getSchoolName(): string {
```

Comment:
This file should be removed and replaced by the correct usage of the new entity concept with DB and non-DB attributes

Reply:
@christophscheuing Shall we do this together in a pair programming session? Two pairs of eyes (yours with the understanding of the previous schools feature and mine or @TheSlimvReal's with experience about the entity schema system) would probably be useful for this.

Reply:
@sleidig In general yes, I'd very much like to do so, but to be honest my understanding of the previous schools feature is rather basic, so I don't really know if this wouldn't rather be a waste of time on your side.